docker/local-development.md

```diff
@@ -95,92 +53,42 @@ start of an application until the services it needs have finished building.
-      - .:/app
-      - ${PWD}/<YOUR_PROJECT>/settings_deployment.py.example:/app/<YOUR_PROJECT>/settings_deployment.py
-    command: <MIGRATION_COMMAND>  # e.g., python manage.py migrate
-
-  postgres:
-    container_name: dedupe-postgres
-    restart: always
-    image: postgres:11
-    healthcheck:
-      test: ["CMD-SHELL", "pg_isready -U postgres"]
-      interval: 10s
-      timeout: 5s
-      retries: 5
-    volumes:
-      # By default, Postgres instantiates an anonymous volume. Use a named
-      # one, so your data persists beyond the life of the container. See this
-      # post for a discussion of the pitfalls of Postgres and anonymous
-      # volumes: https://linuxhint.com/run_postgresql_docker_compose/
-      - <YOUR_APP>-db-data:/var/lib/postgresql/data
-      # Mount our initialization script.
-      - ${PWD}/scripts/init-db.sh:/docker-entrypoint-initdb.d/10-init.sh
-    ports:
-      - 32001:5432
+### 4. `scripts/init-db.sh` (Optional)
 
-volumes:
-  # Declare your named volume for Postgres.
-  <YOUR_APP>-db-data:
-```
+The default Postgres image exposes [a number of environmental variables](https://hub.docker.com/_/postgres/#environment-variables)
+that allow you to define custom behavior for your container without writing
+additional code. Things you can achieve via environmental variables include
+creating your default database, specifying a default user and password, and
+customizing the location of the database files.
 
-### 4. `tests/docker-compose.yml`
+If you need more advanced functionality, e.g., installing a custom Postgres
+extension, the Postgres image provides a harness for executing arbitrary SQL
+and Bash scripts when a container is initialized. [Read more. &raquo;](https://docs.docker.com/samples/library/postgres/#initialization-scripts)
 
-```yaml
-version: '2.4'
+If you define a custom database initialization script, be sure to mount it
+as a volume in your Postgres container in `docker-compose.yml`.
 
-services:
-  app:
-    # Don't restart the service when the command exits
-    restart: "no"
-    environment:
-      # Define any relevant environmental variables here
-      - ...
+```yml
     volumes:
-      # Multi-value fields are concatenated, i.e., this file will be mounted
-      # in addition to the files and directories specified in the root
-      # docker-compose.yml, so we don't need to specify those volumes again
       - ...
-    command: pytest -sxv
+      # Assuming script lives in scripts/init-db.sh
+      - ${PWD}/scripts/init-db.sh:/docker-entrypoint-initdb.d/10-init.sh
+```
+
+***If you would like to use the Postgis extension,*** we recommend using a
+Postgis image, rather than configuring it in an initialization script.
+
+```bash
+#!/bin/bash
+set -e
+
+psql -U postgres -c "CREATE DATABASE <YOUR_DATABASE>"
+psql -U postgres -d <YOUR_DATABASE> -c "CREATE EXTENSION IF NOT EXISTS <YOUR_EXTENSION>"
+# Add any more database initialization commands you may need here
```

Comment:
Can we move this code block above the line starting `If you define a custom database initialization script, be sure to mount it as a volume in your Postgres container in docker-compose.yml`? That's where I expected it when first reading the paragraph.